libvast/vast/system/disk_monitor.hpp

```diff
@@ -27,6 +27,10 @@ struct disk_monitor_state {
   size_t high_water_mark;
   size_t low_water_mark;
 
+  /// How many partitions to delete at once when reaching the high-water
+  /// mark.
+  size_t step_size;
```

Comment:
Instead of duplicating the config, wouldn't it make more sense to just use the `disk_monitor_config` here?

Reply:
Yup, that's much better.